Machine-Teaching-Examples/moab_pomdp_esn.ink

```diff
@@ -0,0 +1,146 @@
+###
+
+# MSFT Bonsai 
+# Copyright 2021 Microsoft
+# This code is licensed under MIT license (see LICENSE for details)
+
+# Moab sample illustrating usage of memory, as a Machine Teaching strategy,
+# for problems with missing state information, i.e. partially observable.
+# Using a history of states and actions can help when a sensor cannot read
+# all the relevant information to describe the dynamics.
+
+# We recommend including state information if you can measure it (velocity),
```

Comment:
Suggested: 
*Note:* This sample includes only the ball position in the observable state, removing velocity. This is for illustration, and not recommended in real problems: if there is information that you know is helpful for solving the problem, like ball velocity for balancing a ball on a plate, include them! Memory helps capture other history-dependent features that are difficulty to explicitly define.